packages/@o3r/core/schematics/module/templates/base/package.json.template

```diff
@@ -11,6 +11,7 @@
     "ng": "<%= runner %> nx",
     "build": "<%= runner %> nx build apis-manager",
     "prepare:build:builders": "<%= runner %> cpy 'collection.json' dist && <%= runner %> cpy 'schematics/**/*.json' dist/schematics",
+    "prepare:compile": "cp-package-json",
```

Comment:
Not sure to understand why doing this process as it is needed only for package with subentries (which may be asked by the generator but not considered as default behavior :S)

Reply:
But it is adding more that boilerplate, it is also adding complexity to Nx rules.
I would really advice to do it only when there is no other choice as workaround for sub entries packages (which should not be the often anymore)

Reply:
that's fine by me, I will keep the PR to update the ng-package.json that is incorrect anyway